libavformat/hls.c

```diff
@@ -1626,8 +1626,8 @@ static int open_segment(struct playlist *v)
             }
             /* If we need to reload the playlist again below (if
              * there's still no more segments), switch to a reload
-             * interval of half the target duration. */
-            reload_interval = v->target_duration / 2;
+             * interval of 15 ms. */
```

Comment:
We may want to investigate overriding `default_reload_interval` to set what this is... as I think on line 1620 we're going to wait at least a segment duration to reload the playlist, we may want to experiment with changing this default value (and make it configurable) as well as the overriding of the reload interval on line 1630.

Reply:
We want the default behaviour to be unchanged I think (so we don't break livemonitor or anything):"
- new config variable, reload_interval, defaults to -1
   - if -1/unset then reload interval = v->target_duration / 2 (old behaviour)
- no change to max_reload

And we should configure max_reload and reload_interval ourselves in our cpp

Reply:
@nvanderzwet-ssimwave Yeah, your proposed changes make sense, much safer.